src/store/modules/ui.store.js

```diff
@@ -276,6 +295,35 @@ export default {
         setCompareSliderActive({ commit }, args) {
             commit('setCompareSliderActive', args)
         },
+        setTooltipPosition({ commit, state }, { tooltipposition, dispatcher }) {
+            const upCasePos = tooltipposition.toUpperCase()
+            if (
+                TooltipPositions[upCasePos] &&
+                state.tooltipPosition !== TooltipPositions[upCasePos]
+            ) {
+                commit('setTooltipPosition', {
+                    position: TooltipPositions[upCasePos],
+                    dispatcher: dispatcher,
+                })
+            } else if (!TooltipPositions[upCasePos]) {
+                commit('setTooltipPosition', {
+                    position: TooltipPositions.DEFAULT,
+                    dispatcher: dispatcher,
+                })
+            }
+        },
+        toggleTooltipPosition({ commit, getters }, { dispatcher }) {
+            if (getters.noTooltip) {
+                commit('setTooltipPosition', {
+                    position: TooltipPositions.DEFAULT,
+                    dispatcher,
+                })
+            } else if (getters.floatingTooltip) {
+                commit('setTooltipPosition', { position: TooltipPositions.FIXED, dispatcher })
+            } else {
+                commit('setTooltipPosition', { position: TooltipPositions.FLOATING, dispatcher })
+            }
+        },
```

Comment:
Doesn't make sense to have a toggle action for a value that is not a boolean flag

Reply:
It made sense in my mind since we were usually 'toggling' between two states, but only using 'setFeatureInfoPosition' in the code will make it easier to understand, especially if one day we implement other ways to display this.